demos/defender/defender_demo_json/defender_demo.c

```diff
@@ -88,6 +88,19 @@
  */
 #define DEFENDER_RESPONSE_REPORT_ID_FIELD_LENGTH    ( sizeof( DEFENDER_RESPONSE_REPORT_ID_FIELD ) - 1 )
 
+/**
+ * @brief The maximum number of times to run the loop in this demo.
+ */
+#ifndef DEFENDER_MAX_DEMO_COUNT
```

Comment:
As this count is only meant for retrying the demo instead of always running the demo for the max count iterations, it would be good to rename the macro across all demos to mention "retry" in them
```suggestion
#ifndef DEFENDER_DEMO_MAX_RETRIES
```

Reply:
Or `DEFENDER_DEMO_MAX_ITERATIONS` would also be good with me

Reply:
The count is not the maximum number of retries, but the maximum number of demo loops. Renaming those macros may not be correct as proposed. I see that that adding a note in the description would be a better choice.